src/testHelpers/setupTests.js

```diff
@@ -12,9 +12,6 @@ const REACT_ERRORS_REGEX = new RegExp(REACT_ERRORS.join('|'));
 
 const { error } = console;
 
-let reactErrorsCountPerTestSuite = 0;
-const reactErrorsLimitPerTestSuite = 6; // The goal is to have zero React errors per suite so keep fixing errors and lowering this limit
-
 const didSuppressWarning = message => {
```

Comment:
I don't really follow what this is doing or why it's needed. can you explain?

Reply:
Suppresses warnings when we want to write tests to violate prop types

Reply:
https://github.com/bbc/simorgh/pull/3525#issuecomment-527404107